config/locales/space/en.yml

```diff
@@ -10,6 +10,8 @@ en:
         Click the 'Add New Section' link below to add at least one Section to this Space.
       config_missing_members_text_html: |
         Click the 'Invitations' link below to invite people to the join this Space.
+      config_missing_entrance_furniture_text_html: |
+        Your Space's Entrance is missing a Gizmo.  Click '%{room_name}' to add a Gizmo.
```

Comment:
Instead of `Click` can we use `Edit '%{room_name}'`?

Reply:
https://github.com/zinc-collective/convene/pull/1599